### ISSUE TITLE
Whitelist domain setup but not blocking non whitelist

### ISSUE THREAD

**Issue by hotrice** [Rob--W/cors-anywhere#190]
I added my white list to originWhiteList: ['somesite.com'] but it will not block any site from not the WhiteList. The comment said "If non-empty", requests not from an origin in this list will be blocked."

**Comment by Rob--W** [comment]
Did you restart the server after applying the change?

Please show how you are starting the server (including any changes that you have made).

**Comment by hotrice** [comment]
Yes, I have tried restarting the server using heroku restart. My local development can still get data even though it not on the originWhiteList.

**Comment by Rob--W** [comment]
> Please show how you are starting the server (including any changes that you have made).

**Comment by hotrice** [comment]
I only add a domain to the originWhiteList and nothing else. originWhiteList['*.somesite.com']. Unless it doesn't like the wildcard subdomain.

**Comment by zgayjjf** [comment]
Same here.

**Comment by singhsansun** [comment]
As I understand it, in order to whitelist a site one has to set the corresponding environment variable, like:
`heroku config:set CORSANYWHERE_WHITELIST=https://example.com`
(correct me if I'm wrong - I don't know what "use" means in the documentation "[...] on port 8080, use:")

**Comment by Rob--W** [comment]
The displayed information in the README section are shell commands (sh, e.g. in bash) to set environment variables and start the server. What you've shown is indeed one of the valid ways to set the environment variable on Heroku (an alternative is to use the web interface to do so).

**Comment by ruslaniv** [comment]
I have tried whitelisting my pc by either using Heroku interface like this:
![Screen Shot 2020-09-17 at 16 23 27](https://user-images.githubusercontent.com/7631314/93452059-2a1cc200-f902-11ea-93ca-fe58fc7f1593.png)
or like this:
![Screen Shot 2020-09-17 at 16 23 40](https://user-images.githubusercontent.com/7631314/93452066-2c7f1c00-f902-11ea-820e-fa8e2b49eaa8.png)
or altering the whitelist directly in the source file like this:
`var originWhitelist = ['111.111.111.111'];`
No matter what, when I try to make a GET request to an API I get 403 (Forbidden) error. 

Is there anything wrong that I'm doing?

**Comment by Rob--W** [comment]
The origin whitelist only takes exact origins. If you have an application running at `http://111.111.111.111:11111`, then you need to put that whole origin in the environment variable. You cannot omit the port or scheme.

**Comment by skifreetony** [comment]
I'm having trouble  setting up the whitelist site as well. Copied the exact URL into the white list and its getting 403 errors when I try visiting the site. i have also tried inputting it as an object ["https://nospoilnba.vercel.app/"]
![Screen Shot 2021-01-17 at 6 51 30 PM](https://user-images.githubusercontent.com/12958581/104867212-38510e00-58f5-11eb-9252-d948f1523d7d.png)

**Comment by Peterhvu** [comment]
@skifreetony if you look at `function parseEnvList(env)` you'll see `return env.split(',');` so your value should be `https://nospoilnba.vercel.app`  **no bracket, no quote.** If you have more app needed to whitelist `https://nospoilnba.vercel.app,https://another.whitelist.app`
